.github/actions/split_version/action.yaml

```diff
@@ -0,0 +1,52 @@
+name: 'Split Version'
+description: 'Splits the inputed version into major, minor, and patch components'
+inputs:
+  version: 
+    description: 'Version to split'
+    required: true
+  release-branch:
+    description: 'Release branch'
+    required: true
+  existing-version:
+    description: 'Existing version'
+    required: true
+outputs:
+  major-minor-version:
+    description: "Major-minor version"
```

Comment:
S: In general these descriptions are just restating the name of the output which doesn't seem helpful.  We should either make them more explanatory or just omit them if we can.